src/core/a-entity.js

```diff
@@ -68,8 +68,18 @@ var proto = Object.create(ANode.prototype, {
    */
   attachedCallback: {
     value: function () {
+      var assetsEl;  // Asset management system element.
+      var self = this;  // Component.
+
       this.addToParent();
       if (this.isScene) { return; }
+
+      // Wait for asset management system to finish before loading.
+      assetsEl = this.sceneEl.querySelector('a-assets');
+      if (assetsEl && !assetsEl.hasLoaded) {
+        assetsEl.addEventListener('loaded', function () { self.load(); });
```

Comment:
What about `assetsEl.addEventListener('loaded', this.load.bind(this));` and we can get rid of self.